pkg/devfile/OWNERS

```diff
@@ -1,4 +1,3 @@
 # See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
 
 approvers:
-- girishramnani
```

Comment:
This file has no approvers, I wonder if it makes sense to have this file at all.

Reply:
Thanks for catching that. Removed the file.